README.adoc

```diff
@@ -0,0 +1,16 @@
+:repository-owner: arduino-libraries
+:repository-name: Arduino_PF1550
+
+= {repository-name} Library for Arduino =
+
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/check-arduino.yml/badge.svg["Check Arduino status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/check-arduino.yml"]
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml/badge.svg["Compile Examples status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml"]
+image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml/badge.svg["Spell Check status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml"]
+
+Library to handle the PMIC (Power Management IC) on the following Arduino boards
+
+- Arduino Portenta H7 (SKU: [ABX00042](https://docs.arduino.cc/hardware/portenta-h7))
+- Arduino Portenta H7 Lite (SKU: [ABX00045](https://docs.arduino.cc/hardware/portenta-h7-lite))
+- Arduino Portenta H7 Lite Connected (SKU: [ABX00046](https://docs.arduino.cc/hardware/portenta-h7-lite-connected))
+- Arduino Nicla Vision (SKU: [ABX00051](https://docs.arduino.cc/hardware/nicla-vision))

```

Comment:
You are using [Markdown](https://www.markdownguide.org/) markup here, but you used the `.adoc` file extension of the [AsciiDoc](https://asciidoc.org/) markup language.

This causes the links to be rendered incorrectly. You can see how it is rendered here:

https://github.com/arduino-libraries/Arduino_PF1550/blob/93d3bd274efd1cca4c5bd2d35132e434a4e4a5ce/README.adoc

So you must pick one language and use the correct markup and file extension for that language. I recommend always using the now standard Markdown instead of AsciiDoc, which maybe had some nice features but clearly has and will fail to achieve any level of widespread adoption.

Reply:
Changed file format to .md.